lighthouse/config/defaults.py

```diff
@@ -215,11 +215,13 @@
 }
 # TODO: make these the generic failure types
 BECKMAN_FAILURE_TYPES = {
```

Comment:
I assume these are used by biosero aswell? Not sure how much work it would be to rename the constant to something more generic like ROBOT_FAILURE_TYPES

Reply:
renamed